website/content/en/docs/upgrading-sdk-version/operator-sdk-versioning.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
We should probably remove this file (in a separate PR perhaps) because it is mostly re-explaining semver.

Reply:
Raised : https://github.com/operator-framework/operator-sdk/issues/3928

If it reaches the consensus that we can remove we can flag good first issue on that.